src/stories/pages/Landingpage/Landingpage.tsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import styles from "./Landingpage.module.css";
+import Menuecard from "../../components/Menuecard/Menuecard";
 
 export default function Landingpage() {
   return (
-    <div className={styles.main}>
-      <div className={styles.headerContainer}>
-        <h1>Visit Rosenheim</h1>
+    <div className={styles.container}>
+      <h1>Visit Rosenheim</h1>
+      <div className={styles.menuecard}>
+        <Menuecard />
       </div>
```

Comment:
You can also give the className directly to your component `Menuecard` like you did with the component `Covid` ;)

Reply:
Hi Lara, thank u, I implemented this good idea straight away. 👍